tools/system_test_jobs.groovy

```diff
@@ -19,39 +20,52 @@ def genJob(driver, platform) {
 
         label(platform)
 
+        parameters {
+            stringParam('sha1', 'master', 'SHA to build')
+        }
+
         scm {
             git {
-                branch('generate_jenkins_jobs')
                 extensions {
                     wipeWorkspace()
                 }
+                branch('${sha1}')
                 remote {
                     github('ni/nimi-python')
-                    credentials('44e8e6ce-9dc2-486e-bf50-9c015febb7bf')
+                    credentials("${credentials_to_use}")
+                    refspec('+refs/pull/${ghprbPullId}/*:refs/remotes/origin/pr/${ghprbPullId}/*')
+                    name('origin')
                 }
             }
         }
 
         triggers {
-            scm('H/3 * * * *')
+            githubPullRequest {
+                admins(['texasaggie97', 'marcoskirsch', 'sbethur'])
+                userWhitelist(['injaleea', 'bhaswath', 'AlexHearnNI'])
```

Comment:
This is a maintenance burden. Since we have the orgWhitelist, can we get rid of userWhitelist?
What special power does being in the admins list give us?

Reply:
Whitelist removed. Admins are who can say to allow a test from a non-allowed person.